test/viaq-data-model.sh

```diff
@@ -215,3 +215,12 @@ if [ -n "$is_maximal" ] ; then
     os::cmd::try_until_failure "oc get pod $muxpod"
     os::cmd::try_until_text "oc get pods -l component=mux" "^logging-mux-.* Running "
 fi
+
+# TEST 6
+# catch logs at the debug level
+TEST_REC_PRIORITY=debug wait_for_fluentd_to_catch_up get_logmessage get_logmessage2
+qs='{"query":{"term":{"systemd.u.SYSLOG_IDENTIFIER":"'"${logmessage2}"'"}}}'
+os::cmd::expect_success "curl_es ${es_ops_pod} /.operations.*/_count -X POST -d '$qs' | \
+                         jq '.count > 0'"
+unset TEST_REC_PRIORITY
```

Comment:
You do not need to unset it.  When you do
```bash
VAR=value command arg ...
```
the `VAR` is set as an environment variable for the command *only*, and nowhere else.

Reply:
ack